main.md

```diff
@@ -169,25 +175,52 @@ This example shows a combined request asking for access to account information a
 
-This field MUST be compared using an exact byte match of the string value against known types by the AS. The AS MUST ensure that there is no collision between different authorization data types that it supports. The AS MUST NOT do any collation or normalization of data types during comparison.
+If an application or API is expected to be deployed across different servers, such as the case in an open standard, the API designer is RECOMMENDED to use a collision-resistant namespace under their control, such as a URI that the API designer controls.
+
+The following example shows how an implementation could utilize the namespace `https://scheme.example.org/` to ensure collision-resistant type values.
+
+```JSON
+{
+   "type": "https://scheme.example.org/files",
+   "locations": [
+      "https://example.com/files"
+   ],
+   "permissions": [
+      {
+         "path": "/myfiles/A",
+         "access": [
+            "read"
+         ]
+      },
+      {
+         "path": "/myfiles/A/X",
+         "access": [
+            "read",
+            "write"
+         ]
+      }
+   ]
+}
+```
+Figure: Example for authorization details with an URL as type identifier.
 
-The value of the `type` field determines the allowable contents of the object which contains it.
+## Common data fields {#common_data_fields}
 
-This draft defines a set of common data elements that are designed to be usable across different types of APIs. These data elements MAY be combined in different ways depending on the needs of the API. All data elements are OPTIONAL for use by a given API definition. The allowable values of all elements are determined by the API being protected.
+This specification defines a set of common data fields that are designed to be usable across different types of APIs. These data fields MAY be combined in different ways depending on the needs of the API. All data fields are OPTIONAL for use by a given API definition. The allowable values of all fields are determined by the API being protected.
 
 `locations`:
 :   An array of strings representing the location of the resource or resource server. These strings are typically URIs identifying the
-    location of the RS.
+    location of the RS. The field `locations` allows a client to specify where it intends to use a certain authorization, i.e., it is  possible to unambiguously assign permissions to resource servers. In situations with multiple resource servers, this prevents unintended client authorizations (e.g. a `read` scope value potentially applicable for an email as well as a cloud service) through audience restriction.
```

Comment:
I would rather see this as a security consideration than in the definition here, but keep a forward reference. So extract the explanation text and leave something like this:

```suggestion
    location of the RS. This field can allow a client to specify a particular RS, as discussed in {{security-considerations-location-field-values}}.
```

Reply:
adopted your proposal